packages/aragon-wrapper/src/ens/index.test.js

```diff
@@ -33,7 +33,7 @@ test('should resolve address for node', (t) => {
   // arrange
   const hackyOptions = {
     provider: {
-      sendAsync: undefined
+      sendAsync: 2
```

Comment:
@0x6431346e Not sure exactly what this is testing (number of calls to `sendAsync`?) but this was failing and changing it to `2` fixes it.

Reply:
It's supposed to test that if you call `resolve` with no `opts`, the `ENS` constructor will be called with:
```
{
  provider: {
    sendAsync: undefined
  }
}
```
and that is because `opts` defaults to `{ provider: {}}` and then, we also assign sendAsync to `provider.send` (which is `undefined`) here:
```
 // Stupid hack for ethjs-ens
  if (!opts.provider.sendAsync) {
    opts.provider.sendAsync = opts.provider.send
  }
``` 
The reason it receives `2` instead of `undefined` is because ava runs tests concurrently now, and these tests share `ethjsEnsStub`...

We can fix this by modifying both tests to `test.serial('should ...`

Reply:
@0x6431346e Actually found a better solution via [`ava`'s contexts](https://github.com/avajs/ava#test-context); see https://github.com/aragon/aragon.js/pull/168/commits/65799d72eaad1269a29506239ac1eca16724430d